### PR TITLE
Add a note for OpenShift deployment.

### LINKED ISSUE #79: Problems to run airbyte-module on OpenShift
When we [deployed](https://github.com/fybrik/airbyte-module/blob/main/fybrik/README_mysql.md#writing-dataset-with-fybrik-application) airbyte-module on OpenShift we have faced with a number of issues:

- The security context was not accepted by the OpenShift
- In order to run dind container, the pod has `privilege: true`  property, which was not acceptable by OpenShift.
- Without the `privilege: true`  property, the docker demon container cannot run.

### PR DESCRIPTION
**Pull request by Mohammad-nassar10**
Fixes #79 

Signed-off-by: mohammad-nassar10 <mohammad.nassar@ibm.com>

### CODE REVIEW

File: fybrik/README.md [hunk 1]
```diff
@@ -52,6 +52,13 @@ You will need a copy of the Fybrik repository (`git clone https://github.com/fyb
    ```bash
    kubectl get pods -n fybrik-blueprints
    ```
+---
+ > _NOTE:_ If you are using OpenShift cluster you will see that the deployment fails because OpenShift doesn't allow `privileged: true` value in `securityContext` field by default. Thus, you should add the service account of the module's deployment to the `privileged SCC` using the following command:
+```bash
+oc adm policy add-scc-to-user privileged system:serviceaccount:fybrik-blueprints:<SERVICE_ACCOUNT_NAME>
+```
+> Then, the doployment will be restarted and creating the pods in `fybrik-blueprints` namespace.
```
Comment (cdoron) [inline fybrik/README.md:60]: doployment -> deployment 
How about: "Then, the deployment will be restarted and pods will be created in the `fybrik-blueprints` namespace."

Comment (roytman) [reply fybrik/README.md:60]: This looks like somebody else (user ?) will restart the deployment. Maybe "The deployment will restart the failed pods and the pods should start successfully."

Comment (Mohammad-nassar10) [reply fybrik/README.md:60]: What about this: "Then, the deployment will be restarted automatically and will restart the failed pods in `fybrik-blueprints` namespace."?

Comment (roytman) [reply fybrik/README.md:60]: The Kubernenes deployment will not be restarted, when deployment cannot achieve the desire number of running pods, **it** periodically restarts the pods.

Comment (Mohammad-nassar10) [reply fybrik/README.md]: Fixed.

**Review by cdoron — APPROVED** [review]

**Pull request merged**